### PR TITLE
String updates

### PR DESCRIPTION
**Pull request by marcellamaki**

### CODE REVIEW

File: kolibri/plugins/device/assets/src/views/DeviceSettingsPage/index.vue [hunk 1]
```diff
@@ -1002,7 +1002,7 @@
       },
       allowLearnersDownloadDescription: {
         message:
-          'Allow users to mark resources for Kolibri to download when available in their network.',
+          'Allow users to explore resources they don’t have and mark it for Kolibri to automatically download when it’s available in their network.',
```
Comment (radinamatic) [inline kolibri/plugins/device/assets/src/views/DeviceSettingsPage/index.vue]: Wrong apostrophe character here? Did we agree on `'`?

Comment (rtibbles) [reply kolibri/plugins/device/assets/src/views/DeviceSettingsPage/index.vue]: I believe we did!

**Review by radinamatic — APPROVED** [review]
Just two apostrophes to turn into `'`, otherwise good to go! 👍🏽

File: kolibri/plugins/device/assets/src/views/DeviceSettingsPage/index.vue (hunk 1 above)
Comment (marcellamaki) [reply kolibri/plugins/device/assets/src/views/DeviceSettingsPage/index.vue]: I will be doing a find and replace for all. These are coming from any strings that are copy/pasted from ditto and figma. So, we have some in these new strings (I should have been more attentive today as I was doing this) and some existing strings as well.

**Review by rtibbles — APPROVED** [review]

**Pull request merged**